pkg/bitcoin/transaction.go

```diff
@@ -238,3 +239,19 @@ type UnspentTransactionOutput struct {
 	// Value denotes the number of unspent satoshis.
 	Value int64
 }
+
+// TransactionMerkleBranch holds information about the merkle branch to
+// a confirmed transaction.
+type TransactionMerkleBranch struct {
```

Comment:
Why don't we call it `TransactionMerkleProof` to be aligned with the Electrum library we use?

Reply:
Done in d44c5c315e8111bf234ae1886725f363b18ecff4.